index.html

```diff
@@ -120,7 +120,7 @@
       <li><a href="https://github.com/Kottans" class="vimeo" target="_blank">Github</a></li>
     </ul>
     <div class="legal">
-      <p> Copyright &copy; 2015 Kottans. All rights reserved.</p>
+      <p> Copyright &copy; <span id="current-year">2016</span> Kottans. All rights reserved.</p>
```

Comment:
We can do inline-way also: 

`<p> Copyright &copy; <script>document.write((new Date).getFullYear());</script> Kottans. All rights reserved.</p>`